_posts/en/newsletters/2020-06-17-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #102'
+permalink: /en/newsletters/2020/06/17/
+name: 2020-06-17-newsletter
+slug: 2020-06-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes the CoinPool payment pool proposal and
+the WabiSabi coordinated coinjoin protocol.  Also included are our
+regular sections with notable changes to services, client software, and
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **CoinPool generalized privacy for identifiable onchain protocols:**
+  Antoine Riard and Gleb Naumenko [posted][coinpool post] to the
+  Bitcoin-Dev mailing list about payment pools, a technique for
+  improving privacy against third-party block chain surveillance by
+  allowing several users to trustlessly share control over a single
+  UTXO.  Compared to previous designs for payment pools (such as
+  [joinpool][]), the CoinPool design focuses on allowing participants to
+  make offchain commitments to transactions between the members of the
+  pool.  Using [taproot][topic taproot], this allows the cooperating
+  participants to operate protocols such as LN or [vaults][topic vaults]
+  using UTXOs that are indistinguishable from single-key UTXOs,
+  improving both participant privacy and onchain scalability.  In that
+  sense, the protocol acts as a sort of generalized [channel
+  factory][topic channel factories] that applies not just to LN but to
+  many protocols that create onchain transactions with unique
+  fingerprints.
+
+    The authors outline how CoinPool could work using existing features
+    of Bitcoin plus taproot, [SIGHASH_NOINPUT][topic sighash_noinput], and the ability to use
+    a delete-only accumulator via Bitcoin Script (e.g. a merkle tree;
+    perhaps something like the proposed [BIP116][]
+    `OP_MERKLEBRANCHVERIFY`).  They don't appear to be advocating a
+    specific design but instead want to start a discussion about how
+    CoinPool or something like it could provide a generalized mechanism
+    that wallets use by default to eliminate the onchain footprint of
+    various current and proposed multi-user protocols.
+
+- **WabiSabi coordinated coinjoins with arbitrary output values:** in
+  the [coinjoin][topic coinjoin] protocol, a group of users
+  collaboratively create a transaction template that spends some of
+  their existing UTXOs (inputs) to a new set of UTXOs (outputs).  The
+  way this transaction template is created has implications for the
+  privacy provided by the coinjoin, and different implementations
+  use different methods:
+
+    <!-- Taker creates tx template, see discussion between harding and
+    waxwing: http://gnusha.org/joinmarket/2020-06-14.log -->
+
+    - [Joinmarket][] has the user paying for the coinjoin (the *market
```

Comment:
I think the sentence structure would be simpler, and easier to parse, as "In Joinmarket, the user paying for the coinjoin (the market taker) contacts ...., collects, ... and creates ...

With "(the market taker)" between "has the user paying for the coinjoin" and "contact each..." its a little difficult to parse that 'the user paying' is the subject of a clause.